pkg/gateway/client/user.go

```diff
@@ -38,7 +38,8 @@ func (c *Client) DeleteUser(ctx context.Context, username string) (*types.User,
 
 		if existingUser.Role.HasRole(types2.RoleAdmin) {
 			var adminCount int64
-			if err := tx.Model(new(types.User)).Where("role = ?", types2.RoleAdmin).Count(&adminCount).Error; err != nil {
+			// We filter out empty email users here, because that is the bootstrap user.
+			if err := tx.Model(new(types.User)).Where("role = ? and email != '' and ", types2.RoleAdmin).Count(&adminCount).Error; err != nil {
```

Comment:
Is there an extra `and` at the end here?

Reply:
Thank you for catching this!